std_msgs/Header header
float32 total_voltage
float32 measured_voltage
float32 current
float32 soc
uint32 capacity
